README.rst

```diff
@@ -64,7 +64,8 @@ To install the development version:
 
     git clone https://github.com/Bachmann1234/diff-cover.git
     cd diff-cover
-    python setup.py install
+    poetry install
+    poetry shell
```

Comment:
Unrelated to the rest of the PR -- I noticed that the instructions mention `setup.py`, which no longer exists.